src/openapi/transport/auth.js

```diff
@@ -155,28 +149,41 @@ TransportAuth.prototype.head = makeTransportMethod('head');
 TransportAuth.prototype.options = makeTransportMethod('options');
 
 /**
- * Run debounced cleanup of error counter map
+ * Cleanup of error counter map
  */
-TransportAuth.prototype.debounceErrorCounterCleanup = function() {
-    if (this.errorCleanupTimoutId) {
-        clearTimeout(this.errorCleanupTimoutId);
+TransportAuth.prototype.cleanupAuthErrors = function() {
+    const cleanThoseBefore = Date.now() - this.authErrorsIgnoreDuration;

```

Comment:
the name is confusing 
we are not actually ignoring the auth errors but keeping them for certain duration to check whether the req with same url with diff authtoken failed ?

Reply:
Tried to improve the names..